Android/app/src/main/java/aschi2403/tsiy/repository/WorkoutRepo.kt

```diff
@@ -179,4 +180,14 @@ class WorkoutRepo(context: Context) {
     fun deleteWorkoutEntry(workoutEntry: WorkoutEntry) {
         workoutEntryDao.deleteWorkoutEntry(workoutEntry)
     }
+
+    fun insertGPSPoints(gpsPoints: List<GPSPoint>) {
+        gpsPoints.forEach {
+            gpsPointsDao.insertGPSPoint(it)
+        }
+    }
+
+    fun getGPSPointsFromActivity(workoutEntryId: Long): List<GPSPoint> {
```

Comment:
you can leave it like this just want to show you something cool:
fun getGPSPointsFromActivity(workoutEntryId: Long): List<GPSPoint> = gpsPointsDao.loadGPSPoints(workoutEntryId)